pkgs/halo2-eth-membership-worker/src/utils.ts

```diff
@@ -27,3 +27,14 @@ export function hexToLittleEndianBytes(hex, size) {
   // Reverse the byte order for little-endian
   return new Uint8Array(bytes).reverse()
 }
+
+// @src https://github.com/RiverRuby/halo2-lib-wasm/blob/9e22a0452add207800a4aa585e345cc9fb50a549/browser/lib/halo2Prover/halo2Prover.ts#L3C1-L11C3
+export const fetchKzgParams = async (k: number) => {
```

Comment:
do you know the valid possible values here?
Or can we predict which k will be used to generate the proofs?

By trying to fetch different files, I could get response for `k` between 11 and 19, (1..18, 20...above fails)
So `k` can't be arbitrary

So should we do a run time check on `k`?

Reply:
> do you know the valid possible values here?
> Or can we predict which k will be used to generate the proofs?

Well that would require still benchmarking first to know exactly which is the best value for K, but for now it will be fixed in the UI as 15; 

> By trying to fetch different files, I could get response for k between 11 and 19, (1..18, 20...above fails)
So k can't be arbitrary
> So should we do a run time check on k?

I can do a check for that first in that function.

Reply:
If we know we only need k=15,
let's just add (commit) this file to the src files.
Then it will part of the deployed files and we don't need to fetch it at all.

We can fine tune and see if we need other k files later.